server/src/lexer/pest/python.pest

```diff
@@ -82,7 +84,10 @@ block_comment_ending = { newline_eoi }
 
 // ## Code blocks
 code_block = { code_line+ }
-code_line = _{ (!doc_block ~ logical_line ~ NEWLINE) | (!doc_block ~ not_newline+ ~ EOI) }
+single_line_string = {("\"" ~ (!("\"") ~ ANY)* ~ "\"") | ("'" ~ (!("'") ~ ANY)* ~ "'")}
+// If there is text or characters within the triple quoted string, it will be regarded as a code block.
```

Comment:
Hints -- a code block is:
- Not a doc block, followed by non-newline characters then a newline OR
- Not a doc block, followed by non-newline characters  then EOI OR
- non-newline characters followed by a newline OR
- a multi-line string